app/controllers/mixins/start_url.rb

```diff
@@ -41,7 +36,6 @@ def start_page_options
 
   def start_page_allowed?(start_page)
     return false if STORAGE_START_PAGES.include?(start_page)
```

Comment:
(Unless this was broken before, or unless `StorageManager` should have been under that setting too)